homeassistant/components/sensor/mercedesme.py

```diff
@@ -0,0 +1,127 @@
+"""
+Support for Mercedes cars with Mercedes ME.
+"""
+import logging
+import datetime
+
+from homeassistant.helpers.entity import Entity
+
+DEPENDENCIES = ['mercedesme']
+
+DOMAIN = 'mercedesme'
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the sensor platform."""
+    if discovery_info is None:
+        return
+
+    controller = hass.data[DOMAIN]['controller']
+
+    if not controller.cars:
+        return False
+
+    for car in controller.cars:
+        add_devices([Sensor('fuelLevelPercent', car, controller)])
+        add_devices([Sensor('fuelRangeKm', car, controller)])
+        add_devices([Sensor('serviceIntervalDays', car, controller)])
+        add_devices([Sensor('odometerKm', car, controller)])
+        add_devices([Sensor('doorsClosed', car, controller)])
+        add_devices([Sensor('windowsClosed', car, controller)])
+        add_devices([Sensor('locked', car, controller)])
+        add_devices([Sensor('tireWarningLight', car, controller)])
+        add_devices([Sensor('latestTrip', car, controller)])
+
+    return True
+
+
+class Sensor(Entity):
+    """Representation of a Sensor."""
+
+    def __init__(self, SensorName, Car, Controller):
+        """Initialize the sensor."""
+        self._state = None
+        self.__name = SensorName
+        self.__car = Car
+        self.controller = Controller
+        self.update()
```

Comment:
Remove this and add [`True`](https://home-assistant.io/developers/code_review_platform/#5-entity) as second parameter to `add_devices([Sensor(...)], True)`